### PR TITLE
Update branding in documentation

### PR DESCRIPTION
**Pull request by peterjunpark**
Related: #428

### CODE REVIEW

File: docs/tutorial/learning-resources.rst [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-Introduction to Omniperf
+ROCm Compute Profiler was previously known as Omniperf. Some of the following
+resources use the earlier name.
+
+Introduction to ROCm Compute Profiler
   :fab:`youtube` `AMD profiling workshop (Pawsey Supercomputing Research Centre) <https://www.youtube.com/watch?v=9AkxBCiInCw>`_
 
-Omniperf example exercises
+ROCm Compute Profiler example exercises
   `<https://github.com/amd/HPCTrainingExamples/tree/main/OmniperfExamples>`__
```
Comment (peterjunpark) [inline docs/tutorial/learning-resources.rst:20]: Will these examples be updated w/ the rename? https://github.com/amd/HPCTrainingExamples/tree/main/Omniperf-OpenMP/Fortran/1_collapse

File: docs/install/grafana-setup.rst [hunk 2]
```diff
@@ -165,12 +165,12 @@ connection is successful.
...
 
-Import the Omniperf dashboard file
-----------------------------------
+Import the ROCm Compute Profiler dashboard file
+-----------------------------------------------
 
 From the **Create** → **Import** page, upload the dashboard file,
 ``/dashboards/Omniperf_v{__VERSION__}_pub.json`` from the
-:doc:`Omniperf tarball <core-install>`.
+:doc:`ROCm Compute Profiler tarball <core-install>`.
```
Comment (peterjunpark) [inline docs/install/grafana-setup.rst:173]: Will the dashboard files be renamed? https://github.com/ROCm/rocprofiler-compute/tree/amd-staging/grafana/dashboards

File: docs/index.rst [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
+behind the tool and its components.
 
-If you're new to Omniperf, familiarize yourself with the tool by reviewing the
+If you're new to ROCm Compute Profiler, familiarize yourself with the tool by reviewing the
 chapters that follow and gradually learn its more advanced features. To get
-started, see :doc:`What is Omniperf? <what-is-omniperf>`.
+started, see :doc:`What is ROCm Compute Profiler? <what-is-rocprof-compute>`.
 
-Omniperf is open source and hosted at `<https://github.com/ROCm/omniperf>`__.
+ROCm Compute Profiler is open source and hosted at `<https://github.com/ROCm/omniperf>`__.
```
Comment (amd-jnovotny) [inline docs/index.rst]: Ref to Omniperf here.

File: docs/install/grafana-setup.rst [hunk 4]
```diff
@@ -158,12 +165,12 @@ connection is successful.
 
 .. _grafana-import-dashboard-file:
 
-Import the Omniperf dashboard file
-----------------------------------
+Import the ROCm Compute Profiler dashboard file
+-----------------------------------------------
 
 From the **Create** → **Import** page, upload the dashboard file,
 ``/dashboards/Omniperf_v{__VERSION__}_pub.json`` from the
```
Comment (amd-jnovotny) [inline docs/install/grafana-setup.rst:172]: Should this still reference Omniperf? Might not be updated.

File: docs/install/grafana-setup.rst [hunk 5]
```diff
@@ -1,26 +1,26 @@
...
-   :keywords: Omniperf, ROCm, profiler, tool, Instinct, accelerator, AMD,
+   :description: ROCm Compute Profiler Grafana server installation and deployment
+   :keywords: ROCm Compute Profiler, ROCm, profiler, tool, Instinct, accelerator, AMD,
               install, deploy, Grafana, server, configuration, GUI
 
-****************************************
-Setting up a Grafana server for Omniperf
-****************************************
+***************************************************
+Setting up Grafana server for ROCm Compute Profiler
```
Comment (amd-jnovotny) [inline docs/install/grafana-setup.rst:7]: Many of the images here still show Omniperf.

File: docs/tutorial/profiling-by-example.rst [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
 Profiling by example
 ********************
 
 The following examples refer to sample :doc:`HIP <hip:index>` code located in
-:fab:`github` :dev-sample:`ROCm/omniperf/blob/amd-mainline/sample <>` and distributed
-as part of Omniperf.
+:fab:`github` :dev-sample:`ROCm/rocprofiler-compute/blob/amd-mainline/sample <>`
+and distributed as part of ROCm Compute Profiler.
 
 .. include:: ./includes/valu-arithmetic-instruction-mix.rst
```
Comment (amd-jnovotny) [inline docs/tutorial/profiling-by-example.rst:13]: Seem to be several references to omniperf commands in the include files here. I couldn't quite track down where they came from (i..e omniperf profile -n instmix --no-roof -- ./instmix)

Comment (peterjunpark) [reply docs/tutorial/profiling-by-example.rst:13]: Updated in a7161d6

File: docs/install/grafana-setup.rst (hunk 4 above)
Comment (peterjunpark) [reply docs/install/grafana-setup.rst:172]: The dashboard files are still called that. See https://github.com/ROCm/rocprofiler-compute/tree/amd-staging/grafana/dashboards.

**Review by amd-jnovotny — APPROVED** [review]
Looks good now.

File: docs/install/grafana-setup.rst (hunk 5 above)
Comment (peterjunpark) [reply docs/install/grafana-setup.rst:7]: I think it's ok to leave those for now

**Comment by peterjunpark** [comment]
@amd-jnovotny Thx for catching those

File: docs/install/grafana-setup.rst (hunk 4 above)
Comment (dgaliffiAMD) [reply docs/install/grafana-setup.rst:172]: @xuchen-amd , are you renaming these files as a part of a subsequent change or will they keep they're name for now?

**Review by dgaliffiAMD — APPROVED** [review]

File: docs/sphinx/static/css/o_custom.css [hunk 7]
```diff
@@ -1,30 +1,8 @@
-:root {
```
Comment (xuchen-amd) [inline docs/sphinx/static/css/o_custom.css:1]: are these deletes expected changes?

File: docs/install/grafana-setup.rst (hunk 4 above)
Comment (xuchen-amd) [reply docs/install/grafana-setup.rst:172]: patch PR created: https://github.com/ROCm/rocprofiler-compute/pull/469

File: docs/sphinx/static/css/o_custom.css (hunk 7 above)
Comment (peterjunpark) [reply docs/sphinx/static/css/o_custom.css:1]: Yeah, those CSS rules were removed in #394. They were a temporary fix due to some changes in https://pydata-sphinx-theme.readthedocs.io/en/stable/. I think #428 accidentally reverted those changes.

**Comment by dgaliffiAMD** [comment]
Cherry picking to https://github.com/ROCm/rocprofiler-compute/pull/468

**Pull request merged**